api/users/serializers.py

```diff
@@ -57,8 +61,10 @@ class UserSerializer(JSONAPISerializer):
     timezone = HideIfDisabled(ser.CharField(required=False, help_text="User's timezone, e.g. 'Etc/UTC"))
     locale = HideIfDisabled(ser.CharField(required=False, help_text="User's locale, e.g.  'en_US'"))
     social = ListDictField(required=False)
-    employment = DevOnly(ser.ListField(child=ser.DictField(), source='jobs', read_only=True))
-    education = DevOnly(ser.ListField(child=ser.DictField(), source='schools', read_only=True))
+    employment = JSONAPIListField(required=False, source='jobs')
+    employment_required_keys = {'department', 'institution', 'ongoing', 'startMonth', 'startYear', 'title'}
+    education = JSONAPIListField(required=False, source='schools')
+    education_required_keys = {'department', 'institution', 'ongoing', 'startMonth', 'startYear', 'degree'}
```

Comment:
You can remove `employment_required_keys` and `education_required_keys` since we're using jsonschema.